gke-nfs-lb/charts/v0.0.1/nfs-csi-lb/templates/csi-nfs-lb-driverinfo.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: nfs.csi.k8s.io
+spec:
+  attachRequired: true
+  volumeLifecycleModes:
+    - Persistent
+  fsGroupPolicy: File
```

Comment:
We might want to disable this. Customer plans 1 PB server, so it would be disasterous if we try to fsgroup.

Reply:
changed to None